capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentImpl.java

```diff
@@ -222,11 +224,21 @@ private void handleQRCodeDetected(@Nullable final PaymentQRCodeData paymentQRCod
             return;
         }
 
-        if (paymentQRCodeData == null) {
-            mUnsupportedQRCodePopup.show(qrCodeContent);
-        } else {
-            mPaymentQRCodePopup.show(paymentQRCodeData);
-        }
+        if (mUnsupportedQRCodePopup.isShown() || mPaymentQRCodePopup.isShown())
+            return;
+
+        showQRCodeView(paymentQRCodeData);
+    }
+
+    private void showQRCodeView(PaymentQRCodeData data) {
+        new Handler(Looper.getMainLooper())
```

Comment:
[Request] Two things I'm missing here to fulfill the requirements in the ticket ([see the first bullet point](https://ginis.atlassian.net/browse/PIA-3279)):
* The delay should only apply to the same QR code. If the QR code changes, then the delay should be ignored.
  When I tested there was a delay before showing the QR code UI for the new QR code.
* When detecting a QR code for the first time, then the delay should be ignored.
  When I tested there was a delay before showing the QR code UI for the first QR code.

Reply:
Ah shit I totally forgot tablets